wbooks/src/app/screens/Library/index.tsx

```diff
@@ -15,6 +16,8 @@ interface Props {
 }
 
 function Library({ navigation }: Props) {
+  const books: Book[] = useSelector((state: RootState) => state.books);
+
```

Comment:
You must use the `useEffect` + `dispatch` to get and set the books in the reducer